src/utils.ts

```diff
@@ -105,3 +105,21 @@ function checkIsDevelopment() {
 export function log(message: any): void {
 	console.log(message);
 }
+
+const _keywords = [
```

Comment:
It feels a bit naff having to maintain this list here; the analysis service is supposed to stop us needing to know things like this.

What happens without this? Wouldn't we just fail the refactor with an error as normal?